src/seaport.ts

```diff
@@ -231,40 +241,41 @@ export class OpenSeaPort {
    * ERC20 tokens on their corresponding Uniswap exchange.
    * Emits the `LiquidateAssets` event when the transaction is prompted.
    * @param param0 __namedParameters Object
-   * @param tokenIds Token IDs of the assets.
-   * @param tokenAddresses Addresses of the tokens' contracts. Must be the same length as `tokenIds`. Each address corresponds with its respective token ID in the `tokenIds` array.
+   * @param assets An array of objects with the tokenId and tokenAddress of each of the assets to bundle together.
    * @param accountAddress Address of the user's wallet
    * @param uniswapSlippageAllowedInBasisPoints The amount of slippage that a user will tolerate in their Uniswap trade; if Uniswap cannot fulfill the order without more slippage, the whole function will revert.
    */
   public async liquidateAssets(
-      { tokenIds, tokenAddresses, accountAddress, uniswapSlippageAllowedInBasisPoints }:
-      { tokenIds: string[];
-        tokenAddresses: string[];
+      { assets, accountAddress, uniswapSlippageAllowedInBasisPoints }:
+      { assets: Asset[];
         accountAddress: string;
         uniswapSlippageAllowedInBasisPoints: number; }
     ) {
 
-    if (!tokenIds || !tokenAddresses || tokenIds.length != tokenAddresses.length) {
-      throw new Error("The 'tokenIds' and 'tokenAddresses' arrays must exist and have the same length.")
+    // If no slippage parameter is provided, use a sane default value
+    const uniswapSlippage = uniswapSlippageAllowedInBasisPoints === 0 ? DEFAULT_WRAPPED_NFT_LIQUIDATION_UNISWAP_SLIPPAGE_IN_BASIS_POINTS : uniswapSlippageAllowedInBasisPoints
```

Comment:
small fix but it'll be nice to have this initialized in the parameter declaration so that the whole parameter is optional instead of required